pkg/bootstrap/stage.go

```diff
@@ -4,20 +4,21 @@ import (
 	"archive/tar"
 	"crypto/sha256"
 	"encoding/hex"
-	"io"
-	"io/ioutil"
-	"os"
-	"path/filepath"
-	"regexp"
-	"strings"
-
 	"github.com/google/go-containerregistry/pkg/authn"
 	"github.com/google/go-containerregistry/pkg/name"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/mutate"
 	"github.com/google/go-containerregistry/pkg/v1/remote"
+	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	"github.com/rancher/rke2/pkg/images"
 	"github.com/sirupsen/logrus"
+	"io"
```

Comment:
I think your goimports, goreturns, (whatever you're using) might not be working. The stdlibs should be fmt'd above the 3rd party.